charts/opensearch-dashboards/values.yaml

```diff
@@ -14,6 +14,36 @@ image:
   tag: ""
   pullPolicy: "IfNotPresent"
 
+startupProbe:
```

Comment:
This will not work if OpenSearch is not allowing all traffic right?

Reply:
The used endpoint is accessible even for anonymous requests.

Reply:
Gotcha